### ISSUE TITLE
Any Questions? button

### ISSUE THREAD

**Issue by Div44** [Div44/IIT_BHU_Gymkhana#2]
In the main taskbar alongside contacts, we can have a button "Any Questions". This tab should open something of the sort of a google doc that notes the name, year, roll no., branch of the asking individual, council to which the question is to be asked, and the question. Then, this question can be sent to the required council.

**Comment by buddingboy** [comment]
yes, we can do such things, "Any question" or "Help" heading can be used. but how this google form data will be sent to required council.

Also I want to confirm that I have shared all files successfully so kindly check the shared project looks like this  http://iitbhu.ac.in/gymkhana_test/index.php 

Also we are expected to avoid the sharing of aforementioned test-link.

**Comment by Div44** [comment]
One easy way can be to use a mail client and mail these queries to the council coordinator. Or we can just make it a link to a common google doc which is accessible by all the councils.

The project looks perfect.

**Comment by buddingboy** [comment]
ok, and what about Science and Technology Council Webpage

**Comment by Div44** [comment]
Nitin is making it.

**Comment by Div44** [comment]
I talked to Nitin yesterday. He is making it and he told me that he has
found a template and is working on it. You should also talk to him about how he is doing?.

**Comment by buddingboy** [comment]
Ok bhaiya, and give me nitin gera mobile number

**Comment by nitingera1996** [comment]
my no is 8272859757
if u r free we can meet today after 8:30 or so

**Comment by nitingera1996** [comment]
two templates that can be worked upon.
I will see if I could come today.
Can't say

**Comment by Div44** [comment]
Gaurav, I have create a google doc and added you to it. Add a button in the
nav bar and directly link it to this form.

**Comment by Div44** [comment]
Nitin has pushed the green club page in the repository. link it to the rest
of the site.

**Comment by buddingboy** [comment]
ok
